packages/widget.vite-plugin.unstable/src/dev-plugin/FoundryWidgetDevPlugin.ts

```diff
@@ -251,3 +286,8 @@ function printSetupPageUrl(server: ViteDevServer) {
     }: ${color.green(setupRoute)}`,
   );
 }
+
+async function getIsCodeWorkspacesMode(): Promise<boolean> {
```

Comment:
From the other comment, if it's possible should we not pass `isCodeWorkspacesMode: boolean` around and instead reuse the existing server parameter with `isCodeWorkspacesMode(server)`?